src/System.Drawing.Primitives/src/System/Drawing/Color.cs

```diff
@@ -644,10 +645,7 @@ public override bool Equals(object obj)
         }
 
         public override int GetHashCode()
-        {
-            return unchecked(value.GetHashCode() ^
-                    state.GetHashCode() ^
-                    knownColor.GetHashCode());
-        }
+            => HashHelpers.Combine(
+                    HashHelpers.Combine(value.GetHashCode(), state.GetHashCode()), knownColor.GetHashCode());
```

Comment:
The original implementation didn't include name and I don't think it is worth hashing. Name is only non null if this color is a custom user color with a name specified. This could only cause a hash collision of unequal Colors if the user has created 2 Colors with the same RGB values and different names.